counterparty-core/counterpartycore/lib/transaction.py

```diff
@@ -195,7 +198,14 @@ def construct(
     exclude_utxos_with_balances=False,
 ):
     # Extract tx_info
-    (source, destinations, data) = tx_info
+    (address_or_utxo, destinations, data) = tx_info
+
+    force_utxo = None
```

Comment:
what does this do?

Reply:
Added a comment in the code to explain.
The idea is that now `*.compose()` function could return an UTXO as source, if so the construct function will force the presence of this utxo in the transaction inputs.